Makefile

```diff
@@ -360,9 +360,7 @@ cf-login:
 .PHONY: setup-performance
 setup-performance:
 	export GINKGO_OPTS="";\
-	export SKIP_TEARDOWN=true;\
 	export NODES=1;\
-	export DEPLOYMENT_NAME="autoscaler-performance";\
```

Comment:
we removed this as we wanted to test it against the "PR_NUM-autoscaler" deployment done by the acceptance tests on this PR. If nothing is specified DEPLOYMENT_NAME assumes the pr number by default. Before merging I would bring back `export DEPLOYMENT_NAME="autoscaler-performance"`